### DISCUSSION TITLE
How can I use a database that I have locally?

### DISCUSSION THREAD

**Discussion by vicontiveros00** [pockethost/pockethost#119] [Q&A]
I've been stuck on this for days, I have a local database that I need to get online with Pocketbase. (I've tried using fly.io but flyctl is incredibly broken and unusable on Windows). How can I use a database that I have locally on Pockethost?

**Comment by benallfree** [comment]
Hi @vicontiveros00, I think I saw your post over on PocketBase's discussion forum too :) 

We don't yet have an upload/download feature, but that's a really interesting idea. I'm releasing a CLI tool in the next release. It would be cool to have a `pockethost backup` and `pockethost restore` that uploaded/downloaded to local.

**Reply by vicontiveros00** [reply]
I managed to find a solution, no worries 👍 You can use something like DB Browser to import data directly to PB locally since it's just SQLite.

**Reply by benallfree** [reply]
I'm not sure I understand. I thought you were asking how to move a local database to PocketHost. Is this a PocketHost question at all? Or just a PocketBase question?
